From 3823379966f15a377c863b0d15694b146854dfd8 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=A9bastien=20Villemot?= <sebastien@dynare.org>
Date: Mon, 2 May 2022 17:03:42 +0200
Subject: [PATCH] Compatibility fix for GCC 12

Ref. dynare#1852

(cherry picked from commit 8c07fb5e4310b2ec2988dc62256e1cf0971c05dd)
---
 src/DataTree.hh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/DataTree.hh b/src/DataTree.hh
index 62da5681..6e9f818d 100644
--- a/src/DataTree.hh
+++ b/src/DataTree.hh
@@ -29,6 +29,7 @@ using namespace std;
 #include <iomanip>
 #include <cmath>
 #include <utility>
+#include <memory>
 
 #include "SymbolTable.hh"
 #include "NumericalConstants.hh"
-- 
GitLab

